packages/app-config-writer/src/preview-config/index.ts

```diff
@@ -14,11 +14,17 @@ import { type ToolsLogger } from '@sap-ux/logger';
  * Corresponding files which are used for the preview are renamed or deleted.
  *
  * @param basePath - base path to be used for the conversion
+ * @param convertTests - indicator if test suite and test runner should be included in the conversion (default: false)
  * @param logger logger to report info to the user
  * @param fs - file system reference
  * @returns file system reference
  */
-export async function convertToVirtualPreview(basePath: string, logger?: ToolsLogger, fs?: Editor): Promise<Editor> {
+export async function convertToVirtualPreview(
+    basePath: string,
+    convertTests: boolean = false,
```

Comment:
This is an incompatible change, luckily the module has still a `0.x.y` version. However, considering the growing number of arguments and to avoid incompatible changes in future, do you want to convert the additional parameters into an options object, like:

```typescript
export async function convertToVirtualPreview(
    basePath: string,
    options: { convertTests: boolean; logger?: ToolsLogger; fs?: Editor }
): Promise<Editor>
```

Reply:
Oh you're right 🙈 I adjusted the code according to your suggestion in https://github.com/SAP/open-ux-tools/pull/2675/commits/83f3e82a66f6e75fc12eb47b94062c4b4d4ebdd2